intel-sgx/enclave-runner/src/usercalls/mod.rs

```diff
@@ -1027,7 +1031,7 @@ impl EnclaveState {
             entry: CoEntry::Initial(main.tcs, argv as _, argc as _, 0, 0, 0),
         };
 
-        let num_of_worker_threads = num_cpus::get();
+        let num_of_worker_threads = if num_cpus::get() == 1 {2} else {num_cpus::get()};
```

Comment:
Could be a good idea to write a brief comment here explaining the reasoning for this "adjustment". (I think I even know the reasoning from our shared meetings, but a new reader might not.)

Reply:
Yes this is out of scope for this ticket. I even think we already created a new ticket specifically for this.

Reply:
yes, i'll remove it